packages/big-design/src/components/Table/Row/Row.tsx

```diff
@@ -40,35 +43,68 @@ const InternalRow = <T extends TableItem>({
   };
 
   const label = isSelected ? `Selected` : `Unselected`;
+  const [isExpanded, setIsExpanded] = useState(false);
```

Comment:
The `Table` component should be stateless by design so we should not be adding in new state to the component. See the `selectable` prop to how we achieve this.

🍹 Also, new changes to the `Table` component should be added to the `StatefulTable` component. This can happen in another PR.